Part2.asm

```diff
@@ -0,0 +1,97 @@
+
+# Name: Hien Vo
+# COMP3410
+# Assigment 1
+# Date: 2/11/15
+
+
+
+
+###############################################
+# .text segment. Assembly instructions go here.
+###############################################
+.data
+
+introduction:	 .asciiz "\ Hien Vo\n Part II In First Assignment\n This program will calculate two numbers in different ways.\n"
+prompt1: .asciiz "\n\n Please enter the first number:"
+prompt2: .asciiz "Please enter the second number"
+result1: .asciiz "Resut of sum:\n"
+result2: .asciiz "\n Resut of difference:\n"
+result3: .asciiz "\n Resut of product:\n"
+
+.text
+
+main:
+
+    #t0-to hold first integer
+    #t1-to hold second integer
+    #t2- used to hold the sum of t$1 and t$2
+
+        #Introduction:
+	la $a0, introduction
+	li $v0, 4
+	syscall # call operating system to perform operation
+	
+	# First number
+	
+        li $v0, 4 #syscall to print string
+        la $a0, prompt1  #call prompt1 above the introruction to print
+        syscall # call operating system to perform operation
+
+        li $v0, 5 #syscall to read an integer
+        syscall
+
+        move $t0, $v0  #move it into $t0
```

Comment:
-5 Accepts input greater than 9.  This allows storing numbers beyond the bounds of the array.